activity/.rpdk-config

```diff
@@ -1,4 +1,5 @@
 {
+    "artifact_type": "RESOURCE",
     "typeName": "AWS::StepFunctions::Activity",
     "language": "java",
     "runtime": "java8",
```

Comment:
we should upgrade our runtime to `java8.al2`, as `java8` is deprecated: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Reply:
will be done in a separate PR for all resources